doc/source/examples/dem/plate-discharge/plate-discharge.rst

```diff
@@ -0,0 +1,496 @@
+
+.. code-block:: text
+
+   subsection model parameters
+     subsection contact detection
+       set contact detection method                = dynamic
+       set dynamic contact search size coefficient = 0.9
+       set neighborhood threshold                  = 1.3
+     end
+     subsection load balancing
+       set load balance method     = dynamic
+       set threshold               = 0.5
+       set dynamic check frequency = 2500
+     end
+     set particle particle contact force method = hertz_mindlin_limit_overlap
+     set rolling resistance torque method       = constant_resistance
+     set particle wall contact force method     = nonlinear
+     set integration method                     = velocity_verlet
+     subsection adaptive sparse contacts
+       set enable adaptive sparse contacts = false
+     end
+   end
+
+
+ASC with Load Balancing Simulation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The only differences between ``plate-discharge_base.prm`` and ``plate-discharge_asc-lb.prm`` are the usage of the ASC method with the load balancing, and the name of the folder for outputs.
+
+Model Parameters
+----------------
+
+Here, we use the ASC with the dynamic load balancing still checking if a load balancing is needed every 2500 iterations with a load threshold of 0.5. In with case, the mobility status of the cells from the adaptive sparse contacts will influence the weight, or the computational contribution, of the cell in the load balancing evaluation. The weight factor of the active cells is 0.7, and the weight factor of the inactive cells is 0.5, while the mobile cells always have a fixed weight factor of 1.
+
+.. code-block:: text
+
+   subsection model parameters
+     subsection contact detection
+       set contact detection method                = dynamic
+       set dynamic contact search size coefficient = 0.9
+       set neighborhood threshold                  = 1.3
+     end
+     subsection load balancing
+       set load balance method     = dynamic_with_sparse_contacts
+       set threshold               = 0.5
+       set dynamic check frequency = 2500
+       set active weight factor    = 0.7
+       set inactive weight factor  = 0.5
+     end
+     set particle particle contact force method = hertz_mindlin_limit_overlap
+     set rolling resistance torque method       = constant_resistance
+     set particle wall contact force method     = nonlinear
+     set integration method                     = velocity_verlet
+     subsection adaptive sparse contacts
+       set enable adaptive sparse contacts = true
+       set granular temperature threshold  = 1e-4
+       set solid fraction threshold        = 0.4
+     end
+   end
+
+
+-----------------------
+Running the Simulations
+-----------------------
+
+Simulations can be launched individually with the executable ``lethe-particles`` and the parameter files with the logging of the display in the terminal.
```

Comment:
I don't understand the part ` with the logging of the display in the terminal.`

Reply:
I rephrased it